docs/advanced/general/using-private-git.md

```diff
@@ -16,18 +17,35 @@ E-mail: github@vapor.codes
 
 Link to profile: https://github.com/vapor-cloud-api
 
-### Use SSH Key
+## Add your SSH Key to Vapor Cloud
+
+!!! info
+    This will not be online before September 17 23:45 UTC. You can already now send your key in, and it will be onlien when we roll the system out.
+
+!!! info
+    SSH Keys will be linked to the organization, so they are automatically available to all users and projects under your organization.
+
+### Add your SSH Key.
+
+As this system is still in beta, you need to send your key to our support system, and they will manually add it to the system. Be aware there can be a slight delay in adding the key. You will get a reply once it's added.
+
+We will later create an automatic system to manage the keys.
+
+Send an Email to `support@vapor.cloud`
+
+**Subject:** Add SSH Key: **[Organization name]**
+
+Attach the SSH key as a file to the Email.
 
-You can use the same SSH Key as in 'Another git provider'. This can be setup on either your own user or a user you create yourself.
+If you don't have an SSH Key, you can see how to create a new one [here](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/)
 
-## Another git provider (Gitlab, Bitbucket or self hosted)
+The file is usually located at `~/.ssh/id_rsa`.
```

Comment:
id_rsa is the *private* key. 
`~/.ssh/id_rsa.pub` is the public key

Reply:
We need the private key, to get access to the Git repo :) .. It's the one they need to send us